### DISCUSSION TITLE
<button> and <textarea> not adapting to light theme

### DISCUSSION THREAD

**Discussion by Namyalg** [pradyunsg/furo#444] [Potential Issue] [answered]
I am embedding raw html as 

```
.. raw:: html
   
   <div style="text-align: center;" class="feedback">
      <h3> Was the documentation helpful ? </h3>
         <h4 id="choice-feedback-status"></h4>
            <div id="choice-feedback">
                <span><button onclick="yes()">Yes</button></span>
                <button onclick="no()">No</button><br>
            </div>
            <br>
            <button onclick="showFeedback()">Send Feedback</button>
            <h4 id="text-feedback-status"></h4>
            <div id="text-feedback" style="display: none;">
                <textarea id="message" name="message" rows="4" cols="60"></textarea>
                <br>
                <br>
                <button onclick="saveFeedback()">Submit feedback</button>
            </div>
        </div>
```

However, the `<button>` and `<textarea>` tags do not adapt to light theme. They remain the same with both light and dark themes as :

Light theme:
![image](https://user-images.githubusercontent.com/53875297/169989592-25a26d71-d78c-42dc-b2eb-74649151d496.png)

Dark theme: 
![image](https://user-images.githubusercontent.com/53875297/169989701-87fe09e1-6de8-4b09-a546-6e029a24a860.png)

**Comment by pradyunsg** [comment]
Their color is coming from the browser, which is likely picking them up from the OS theme. Are you on Linux? If yes, this is likely a behaviour derived from your desktop environment.

These are not stylised in Furo since there are no out-of-the-box directives for this.

**Comment by Namyalg** [comment]
Alright, so the best way is to write custom CSS for the desired behavior

**Comment by pradyunsg** [comment, marked as answer]
FWIW, https://pradyunsg.me/furo/kitchen-sink/sphinx-design/#buttons now has stylised buttons that are gracefully supported.
